packages/yoco/svg/catalog2.svg

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg width="100%" height="100%" viewBox="0 0 48 48" version="1.1" xmlns="http://www.w3.org/2000/svg"
+	xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" xmlns:serif="http://www.serif.com/"
+	fill-rule="evenodd" clip-rule="evenodd" stroke-linecap="round">
+	<path
+		d="M23,8C23,6.896 22.104,6 21,6L9,6C7.896,6 7,6.896 7,8L7,20C7,21.104 7.896,22 9,22L21,22C22.104,22 23,21.104 23,20L23,8ZM11,10L11,18L19,18L19,10L11,10Z" />
+	<path
+		d="M23,27C23,25.896 22.104,25 21,25L9,25C7.896,25 7,25.896 7,27L7,39C7,40.104 7.896,41 9,41L21,41C22.104,41 23,40.104 23,39L23,27ZM11,29L11,37L19,37L19,29L11,29Z" />
+	<path d="M27,8L39,8" fill="none" fill-rule="nonzero" stroke="black" stroke-width="4px" />
+	<path d="M27,27L39,27" fill="none" fill-rule="nonzero" stroke="black" stroke-width="4px" />
+	<path d="M27,14L39,14" fill="none" fill-rule="nonzero" stroke="black" stroke-width="4px" />
+	<path d="M27,33L39,33" fill="none" fill-rule="nonzero" stroke="black" stroke-width="4px" />
+	<path d="M27,20L33,20" fill="none" fill-rule="nonzero" stroke="black" stroke-width="4px" />
+	<path d="M27,39L33,39" fill="none" fill-rule="nonzero" stroke="black" stroke-width="4px" />
+</svg>
```

Comment:
Your editorconfig seems to be malfunctioning. There’s no final newline here which is invalid on UNIX-based systems